distribution/docs/src/main/resources/content/_quickstart/quickstart-installing.adoc

```diff
@@ -147,3 +147,53 @@ This installation directory will be referred to as `${home_directory}`.
 ----
 ${branding-lowercase}${at-symbol}local>
 ----
+
+==== Installing Multiple ${branding}s on the Same Host
+
+To have multiple ${branding} instances on the same host, it is necessary to edit the port numbers in the files in the ${branding} install folder.
+
+[cols="5*.^" options="header"]
+|===
+
+|File to Edit
+|Property
+|Original Value
+|Example of New Value
+|Notes
+
```

Comment:
Do we need to update etc/hazelcast.xml, etc/jetty.xml, and etc/org.apache.karaf.shell.cfg?